### ISSUE TITLE
HTML labels present in view mode

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#3089]
[+1 from customer](https://3.basecamp.com/3600924/buckets/2016723/messages/367344485)

**Comment by ebruchez** [comment]
Also look at extra markup which includes lots of `<span>` elements.

**Comment by ebruchez** [comment]
Analysis from Controls form:

- P1
  - `<label for="...">`
    - `for` doesn't refer to any `id`
    - there shouldn't be a `<label>` as there are no HTML form controls (just use a `<span>`)
  - hints are hidden via CSS, but they should probably not be output in the first place
    - especially empty hints
  - same for alerts
  - same for help
- P2
  - would it be better if instead of:
    ```xml
    <div class="xbl-component xbl-fr-grid">
        <div class="fr-grid fr-grid-2 fr-norepeat">
    ```
    we had:
    ```xml
    <div class="xbl-component xbl-fr-grid fr-grid fr-grid-2 fr-norepeat">
    ```
- misc
  - `fr-form-group` should be a `div`
